assets/app/styles/_core.less

```diff
@@ -331,6 +331,96 @@ code {
   overflow-x: auto; // remove when we update to latest patternfly with twbs 3.3.4
 }
 
+.table-events {
+  background-color: #fff;
+  > tbody {
+    + tbody {
+      border-top-width: 1px;
+    }
+  }
+  tr {
+    th,td {
+      .column-combine {
+        display: none;
+      }
+    }
+    @media (min-width: @screen-lg-min) {
+      td[data-title="Name"] {
+        white-space: nowrap;
+      }
+    }
+  }
+  // Combine event reason and message into one column between 768 and 1100, so that table doesn't cause horizontal scroll and message is readable
+  @media (min-width: @screen-sm-min) and (max-width: @screen-md-max) {
+    > thead {
+      > tr {
+        > th {
+          &:nth-child(4) {
+            display: none;
+          }
+          &:nth-child(5) {
+            .column-combine {
+              display: inline;
```

Comment:
This is clever, but I think it might be simpler just to use `visible-*` or `hidden-*` Bootstrap classes to hide the column, and then have a `span` with the same class in the messages column. Then you could do some additional styling if you wanted or add a hyphen or colon to separate the values when combined.